src/test/kotlin/uk/gov/dluhc/registercheckerapi/messaging/InitiateRegisterCheckMessageListenerIntegrationTest.kt

```diff
@@ -29,7 +31,11 @@ internal class InitiateRegisterCheckMessageListenerIntegrationTest : Integration
     @Test
     fun `should process message received on queue`() {
         // Given
-        val message = buildInitiateRegisterCheckMessage()
+        val message = buildInitiateRegisterCheckMessage(
+            personalDetail = buildRegisterCheckPersonalDetail(
+                address = buildRegisterCheckAddress(street = "")
```

Comment:
don't think we need to change this integration test, otherwise this test becomes specific to empty street

Reply:
2 tests? 1 with, 1 without?